modules/auxiliary/scanner/http/wordpress_cp_calendar_sqli.rb

```diff
@@ -0,0 +1,74 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'uri'
+require 'msf/core'
+
+class Metasploit4 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
Not sure what this would buy me, don't need fingerprinting or authentication... Can certainly add if there is a benefit though.